app/src/main/java/cmput301f17t13/com/catisadog/models/FollowRequest.java

```diff
@@ -0,0 +1,20 @@
+package cmput301f17t13.com.catisadog.models;
+
+import org.joda.time.DateTime;
+
+import java.util.UUID;
+
+/**
+ * Created by James on 2017-10-20.
+ */
+
+public class FollowRequest {
```

Comment:
Classes like these are a great way to get into Kotlin. 

https://kotlinlang.org/docs/reference/data-classes.html

Reply:
As much as I'd like to get into Kotlin, the class demands Vanilla java

Reply:
Instead, we will be using https://github.com/google/auto/tree/master/value

Reply:
Aww man, sorry to hear about the Kotlin Embargo :|